config/vlan.py

```diff
@@ -60,6 +60,11 @@ def del_vlan(db, vid):
     
     if keys:
         ctx.fail("VLAN ID {} can not be removed. First remove all members assigned to this VLAN.".format(vid))
+
+    vxlan_table = db.cfgdb.get_table('VXLAN_TUNNEL_MAP')
```

Comment:
why do we have to set this restriction? will there be any ASIC failures?

Reply:
it is leaving behind vxtunnel maps in the configdb to vlans that have been removed. its messy